test/test_type.jl

```diff
@@ -497,6 +497,53 @@ isdir(outdir) && rm(outdir, recursive = true)
             @test eltype(@inferred energy_internal(cons, equations)) == RealT
         end
     end
+
+    @timed_testset "Compressible Euler Quasi 1D" begin
+        for RealT in (Float32, Float64)
+            equations = @inferred CompressibleEulerEquationsQuasi1D(RealT(1.4))
+            x = SVector(zero(RealT))
+            t = zero(RealT)
+            u = u_ll = u_rr = SVector(one(RealT), one(RealT), one(RealT), one(RealT))
+            orientation = 1
+            normal_direction = normal_ll = normal_rr = SVector(one(RealT))
+
+            @test eltype(@inferred initial_condition_convergence_test(x, t, equations)) ==
+                  RealT
+            @test eltype(@inferred source_terms_convergence_test(u, x, t, equations)) ==
+                  RealT
+
+            @test eltype(@inferred flux(u, orientation, equations)) == RealT
+            @test eltype(@inferred flux_nonconservative_chan_etal(u_ll, u_rr, orientation,
+                                                                  equations)) == RealT
+            @test eltype(@inferred flux_nonconservative_chan_etal(u_ll, u_rr,
+                                                                  normal_direction,
+                                                                  equations)) ==
+                  RealT
+            @test eltype(@inferred flux_nonconservative_chan_etal(u_ll, u_rr, normal_ll,
+                                                                  normal_rr, equations)) ==
+                  RealT
+            if RealT == Float32
+                # check `ln_mean` and `inv_ln_mean` (test broken)
+                @test_broken eltype(flux_chan_etal(u_ll, u_rr, orientation, equations)) ==
+                             RealT
+            else
+                @test eltype(@inferred flux_chan_etal(u_ll, u_rr, orientation, equations)) ==
+                      RealT
+            end
+
+            @test eltype(@inferred max_abs_speed_naive(u_ll, u_rr, orientation, equations)) ==
+                  RealT
+            @test eltype(@inferred Trixi.max_abs_speeds(u, equations)) == RealT
+            @test eltype(@inferred cons2prim(u, equations)) == RealT
+            @test eltype(@inferred prim2cons(u, equations)) == RealT
+            # TODO: There is a bug in the `entropy` function
+            # @test eltype(@inferred entropy(u, equations)) == RealT
```

Comment:
Please check this bug - who would be responsible for it?

Reply:
https://github.com/trixi-framework/Trixi.jl/blob/866530092e95332b2fd51a7063d3140ba72a577c/src/equations/compressible_euler_quasi_1d.jl#L315-L321

The `entropy` for `CompressibleEulerEquations1D` do not return as a vector. Who was doing the code review?Seriously this is a apparent bug :(